### DISCUSSION TITLE
List of supported models?

### DISCUSSION THREAD

**Discussion by sunPi** [MAIF/shapash#485] [General]
Just a quick question (i didnt manage to go through all the docs), is there a list of all models that are support for the Shapash smart explainer and predictor? Its probably in your documentation but I gave a quick look and I couldn't find any such information.

**Comment by ThomasBouche** [comment]
Hi, 

Shapash works for Regression, Binary Classification or Multiclass problem.
It is compatible with many models: Catboost, Xgboost, LightGBM, Sklearn Ensemble, Linear models, SVM.

it's noted in the readme, but needs to be better documented.
